bindings/nodejs/types/transactionOptions.ts

```diff
@@ -59,3 +59,15 @@ export interface NftOptions {
     /** Hex encoded bytes */
     metadata?: string;
 }
+
+/** Options for the transaction creation */
```

Comment:
Idk if it's a failed copy/paste or if you wanted the same comment as TransactionOptions, ignore me if yes